lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,26 @@
+module TicTacToe
+end
+
+require_relative "./tic_tac_toe/core_extensions.rb"
+require_relative "./tic_tac_toe/board.rb"
+require_relative "./tic_tac_toe/cell.rb"
+require_relative "./tic_tac_toe/game.rb"
+require_relative "./tic_tac_toe/player.rb"
+
+class Instruct
```

Comment:
That should be into the executable 
Check that link please https://guides.rubygems.org/make-your-own-gem/#adding-an-executable

Reply:
Updated.